packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -195,19 +196,18 @@ export default class RouterService extends Service {
           // Clipboard now has /author/tomster/books
         }
       }
-    });
-    ```
```

Comment:
Probably shouldn't be dropped (I think it will make the next prose section unhappy)

Reply:
Fixed, thanks!